keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceEvent.kt

```diff
@@ -404,7 +430,9 @@ data class ResourceCheckUnresolvable(
   override val version: Int,
   override val application: String,
   override val timestamp: Instant,
-  override val message: String?
+  override val message: String?,
+  override val level: EventLevel = EventLevel.WARNING,
+  override val displayName: String = "Failed to resolve state diff",
```

Comment:
You're using "difference" and "diff" - do you mean to use both forms? do you think it would be better to pick just one? I'm not sure.